### ISSUE TITLE
Not compatible with flake8 >= 3.8.0

### ISSUE THREAD

**Issue by jreese** [ambv/flake8-pyi#37]
```
jreese@jreese-mbp ~/scratch » python3.8 -m venv flake8-venv
jreese@jreese-mbp ~/scratch » source flake8-venv/bin/activate
(flake8-venv) jreese@jreese-mbp ~/scratch » pip install flake8==3.8.1 flake8-pyi==19.3.0
Collecting flake8==3.8.1
  Using cached https://files.pythonhosted.org/packages/bf/47/36e51603431e1a5289eb41636199d2c225fcb1ca286e29c02d219c8e6e88/flake8-3.8.1-py2.py3-none-any.whl
Collecting flake8-pyi==19.3.0
  Using cached https://files.pythonhosted.org/packages/cb/77/894472548bbf67ff3a4b9d2208b8a7beb58414112ad77b7d96a792a7d7c5/flake8_pyi-19.3.0-py2.py3-none-any.whl
Collecting mccabe<0.7.0,>=0.6.0 (from flake8==3.8.1)
  Using cached https://files.pythonhosted.org/packages/87/89/479dc97e18549e21354893e4ee4ef36db1d237534982482c3681ee6e7b57/mccabe-0.6.1-py2.py3-none-any.whl
Collecting pycodestyle<2.7.0,>=2.6.0a1 (from flake8==3.8.1)
  Using cached https://files.pythonhosted.org/packages/10/5b/88879fb861ab79aef45c7e199cae3ef7af487b5603dcb363517a50602dd7/pycodestyle-2.6.0-py2.py3-none-any.whl
Collecting pyflakes<2.3.0,>=2.2.0 (from flake8==3.8.1)
  Using cached https://files.pythonhosted.org/packages/69/5b/fd01b0c696f2f9a6d2c839883b642493b431f28fa32b29abc465ef675473/pyflakes-2.2.0-py2.py3-none-any.whl
Collecting attrs (from flake8-pyi==19.3.0)
  Using cached https://files.pythonhosted.org/packages/a2/db/4313ab3be961f7a763066401fb77f7748373b6094076ae2bda2806988af6/attrs-19.3.0-py2.py3-none-any.whl
Installing collected packages: mccabe, pycodestyle, pyflakes, flake8, attrs, flake8-pyi
Successfully installed attrs-19.3.0 flake8-3.8.1 flake8-pyi-19.3.0 mccabe-0.6.1 pycodestyle-2.6.0 pyflakes-2.2.0
```
```
(flake8-venv) jreese@jreese-mbp ~/scratch » flake8 --version
Traceback (most recent call last):
  File "/Users/jreese/scratch/flake8-venv/bin/flake8", line 10, in <module>
    sys.exit(main())
  File "/Users/jreese/scratch/flake8-venv/lib/python3.8/site-packages/flake8/main/cli.py", line 22, in main
    app.run(argv)
  File "/Users/jreese/scratch/flake8-venv/lib/python3.8/site-packages/flake8/main/application.py", line 360, in run
    self._run(argv)
  File "/Users/jreese/scratch/flake8-venv/lib/python3.8/site-packages/flake8/main/application.py", line 347, in _run
    self.initialize(argv)
  File "/Users/jreese/scratch/flake8-venv/lib/python3.8/site-packages/flake8/main/application.py", line 329, in initialize
    self.register_plugin_options()
  File "/Users/jreese/scratch/flake8-venv/lib/python3.8/site-packages/flake8/main/application.py", line 165, in register_plugin_options
    self.check_plugins.register_options(self.option_manager)
  File "/Users/jreese/scratch/flake8-venv/lib/python3.8/site-packages/flake8/plugins/manager.py", line 494, in register_options
    list(self.manager.map(register_and_enable))
  File "/Users/jreese/scratch/flake8-venv/lib/python3.8/site-packages/flake8/plugins/manager.py", line 302, in map
    yield func(self.plugins[name], *args, **kwargs)
  File "/Users/jreese/scratch/flake8-venv/lib/python3.8/site-packages/flake8/plugins/manager.py", line 490, in register_and_enable
    call_register_options(plugin)
  File "/Users/jreese/scratch/flake8-venv/lib/python3.8/site-packages/flake8/plugins/manager.py", line 402, in generated_function
    return method(optmanager, *args, **kwargs)
  File "/Users/jreese/scratch/flake8-venv/lib/python3.8/site-packages/flake8/plugins/manager.py", line 216, in register_options
    add_options(optmanager)
  File "/Users/jreese/scratch/flake8-venv/lib/python3.8/site-packages/pyi.py", line 359, in add_options
    option.to_optparse().default = option.default
  File "/Users/jreese/scratch/flake8-venv/lib/python3.8/site-packages/flake8/options/manager.py", line 332, in to_optparse
    raise AttributeError("to_optparse: flake8 now uses argparse")
AttributeError: to_optparse: flake8 now uses argparse
```

**Comment by JelleZijlstra** [comment]
Fixed that today! #35

**Comment by jreese** [comment]
Sorry I missed that. Thank you!
